handlers/producers.go

```diff
@@ -152,8 +152,25 @@ func (umh ProducersHandler) CreateProducer(c *gin.Context) {
 		c.AbortWithStatusJSON(500, gin.H{"message": "Server error"})
 		return
 	}
+	var user models.User
```

Comment:
the user can be reached from the token please get it from there instead from the DB